index.html

```diff
@@ -1915,9 +1915,12 @@ <h4 id="maturity-levels">6.1.2 Maturity Levels</h4>
               corresponds to the "Last Call Working Draft" discussed in the Patent Policy.</li>
           </ul>
         </dd>
-        <dd><strong>Note</strong>: Candidate Recommendations are expected to be acceptable as Recommendations. Announcement of a
-          different next step <em class="rfc2119">should</em> include the reasons why the change in expectations comes at so late
-          a stage.</dd>
+        <dd><strong>Note</strong>: Advancing to Candidate Recommendation indicates that
+          no further improvement is expected without additional implementation experience or feedback from the wider community.
```

Comment:
My reading of the Process is that wide review feedback is welcome during CR, and is not restricted to WD prior to CR; it is true that wide review must be demonstrated to enter CR, but that doesn't restrict the opportunity for providing review comments.

Reply:
My reading of the Process is that wide review feedback is welcome during CR, and is not restricted to WD prior to CR; it is true that wide review must be demonstrated to enter CR, but that doesn't restrict the opportunity for providing review comments.

Reply:
My reading of the Process is that wide review feedback is welcome during CR, and is not restricted to WD prior to CR; it is true that wide review must be demonstrated to enter CR, but that doesn't restrict the opportunity for providing review comments.

Reply:
My reading of the Process is that wide review feedback is welcome during CR, and is not restricted to WD prior to CR; it is true that wide review must be demonstrated to enter CR, but that doesn't restrict the opportunity for providing review comments.

Reply:
My reading of the Process is that wide review feedback is welcome during CR, and is not restricted to WD prior to CR; it is true that wide review must be demonstrated to enter CR, but that doesn't restrict the opportunity for providing review comments.

Reply:
My reading of the Process is that wide review feedback is welcome during CR, and is not restricted to WD prior to CR; it is true that wide review must be demonstrated to enter CR, but that doesn't restrict the opportunity for providing review comments.

Reply:
My reading of the Process is that wide review feedback is welcome during CR, and is not restricted to WD prior to CR; it is true that wide review must be demonstrated to enter CR, but that doesn't restrict the opportunity for providing review comments.

Reply:
Right. But this is talking about why we transition to CR. We don't do it to get wide review, we do it because we need implementation experience.

Reply:
OK, but feedback may need to be addressed whether it is implementation-related or not. For an example, if since entering CR some law were passed that had implications for a feature in some country, the WG should probably have a discussion about that.

Reply:
Of course. But this sentence is explaining what distinguishes a CR from a WD, not explaining things that are common to both states.

Reply:
@fantasai I took most of your suggested improvements, except for `s/is expected/can be expected/`, as I believe this risks forbidding leaving things intentionally undefined.